src/components/body.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTumblr } from '@fortawesome/free-brands-svg-icons';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { faQuoteLeft } from '@fortawesome/free-solid-svg-icons';
+
+const arrayQuotes = [
```

Comment:
let's create a file called `array-quotes.js` and export the variable called arrayQuotes, then import it in here.